frontend/components/panels/DesktopClassPanel.js

```diff
@@ -12,11 +12,13 @@ import globe from './globe.svg';
 import desktopCss from './DesktopClassPanel.css';
 import baseCss from './BaseClassPanel.css';
 import macros from '../macros';
-import Keys from '../../../common/Keys';
-import LocationLinks from './LocationLinks';
-import WeekdayBoxes from './WeekdayBoxes';
 import BaseClassPanel from './BaseClassPanel';
+import DesktopSectionPanel from './DesktopSectionPanel';
 
+
+// Merge the base css and the css specific to desktop panels
+// The identityObjProxy check is so the class names appear in the snap files in testing
+// Lets move to sass instead of css eventually.
```

Comment:
How would our stack change? I'm quite familiar with Sass, so I could do that if it's easy to implement.

Reply:
We'd have to add a sass loader to the webpack config and then rename all of the css files to scss files. We could then do `@imports` in the css files instead of this hack in the JS files. Hopefully its not too much work! I'm down to do this in a future PR if you want to take a stab at it. There's also a bunch of other features we could add too.

Reply:
See #40.